client/my-sites/site-settings/media-settings/index.jsx

```diff
@@ -74,13 +74,12 @@ const MediaSettings = ( {
 					disabled={ isRequestingSettings || isSavingSettings }
 					/>
 				<div className="media-settings__module-settings site-settings__child-settings">
-					<FormToggle
-						className="media-settings__carousel-module-settings-toggle is-compact"
+					<CompactFormToggle
 						checked={ fields.carousel_display_exif || false }
 						disabled={ isRequestingSettings || isSavingSettings || ! carouselActive }
 						onChange={ handleAutosavingToggle( 'carousel_display_exif' ) } >
 						{ translate( 'Show photo metadata in carousel, when available' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Show photo metadata in carousel, when available.' )` *ES Score: 19*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/cleanup-site-settings-form-toggles)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).